preparing patch to upstream

skrashevich opened the pull request.

kapustkin committed:
Update NET to 7

skrashevich committed:
Merge pull request #1 from kapustkin/master

Update NET to 7

skrashevich committed:
v10.7.5.6

Upgrade to .NET 6

dependabot[bot] committed:
Bump Newtonsoft.Json in /src/KinopoiskUnofficialInfo.ApiClient

Bumps [Newtonsoft.Json](https://github.com/JamesNK/Newtonsoft.Json) from 12.0.3 to 13.0.1.
- [Release notes](https://github.com/JamesNK/Newtonsoft.Json/releases)
- [Commits](https://github.com/JamesNK/Newtonsoft.Json/compare/12.0.3...13.0.1)

---
updated-dependencies:
- dependency-name: Newtonsoft.Json
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

skrashevich committed:
Create dependabot.yml

dependabot[bot] committed:
Bump xunit from 2.4.0 to 2.4.2 in /src

Bumps [xunit](https://github.com/xunit/xunit) from 2.4.0 to 2.4.2.
- [Release notes](https://github.com/xunit/xunit/releases)
- [Commits](https://github.com/xunit/xunit/compare/2.4...2.4.2)

---
updated-dependencies:
- dependency-name: xunit
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump Microsoft.Extensions.Logging from 5.0.0 to 7.0.0 in /src

Bumps [Microsoft.Extensions.Logging](https://github.com/dotnet/runtime) from 5.0.0 to 7.0.0.
- [Release notes](https://github.com/dotnet/runtime/releases)
- [Commits](https://github.com/dotnet/runtime/compare/v5.0.0...v7.0.0)

---
updated-dependencies:
- dependency-name: Microsoft.Extensions.Logging
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump Moq from 4.16.1 to 4.18.4 in /src

Bumps [Moq](https://github.com/moq/moq4) from 4.16.1 to 4.18.4.
- [Release notes](https://github.com/moq/moq4/releases)
- [Changelog](https://github.com/moq/moq4/blob/main/CHANGELOG.md)
- [Commits](https://github.com/moq/moq4/compare/v4.16.1...v4.18.4)

---
updated-dependencies:
- dependency-name: Moq
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump Newtonsoft.Json from 12.0.3 to 13.0.2 in /src

Bumps [Newtonsoft.Json](https://github.com/JamesNK/Newtonsoft.Json) from 12.0.3 to 13.0.2.
- [Release notes](https://github.com/JamesNK/Newtonsoft.Json/releases)
- [Commits](https://github.com/JamesNK/Newtonsoft.Json/compare/12.0.3...13.0.2)

---
updated-dependencies:
- dependency-name: Newtonsoft.Json
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

skrashevich committed:
Create pull.yaml

skrashevich committed:
Merge branch 'master' into dependabot/nuget/src/Microsoft.Extensions.Logging-7.0.0

skrashevich committed:
Merge pull request #4 from skrashevich/dependabot/nuget/src/Microsoft.Extensions.Logging-7.0.0

Bump Microsoft.Extensions.Logging from 5.0.0 to 7.0.0 in /src

dependabot[bot] committed:
Bump xunit.runner.visualstudio from 2.4.0 to 2.4.5 in /src

Bumps [xunit.runner.visualstudio](https://github.com/xunit/visualstudio.xunit) from 2.4.0 to 2.4.5.
- [Release notes](https://github.com/xunit/visualstudio.xunit/releases)
- [Commits](https://github.com/xunit/visualstudio.xunit/commits)

---
updated-dependencies:
- dependency-name: xunit.runner.visualstudio
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump coverlet.collector from 1.2.0 to 3.2.0 in /src

Bumps [coverlet.collector](https://github.com/coverlet-coverage/coverlet) from 1.2.0 to 3.2.0.
- [Release notes](https://github.com/coverlet-coverage/coverlet/releases)
- [Commits](https://github.com/coverlet-coverage/coverlet/compare/v1.2.0...v3.2.0)

---
updated-dependencies:
- dependency-name: coverlet.collector
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Update System.Text.Json requirement from 6.* to 7.* in /src

Updates the requirements on [System.Text.Json](https://github.com/dotnet/runtime) to permit the latest version.
- [Release notes](https://github.com/dotnet/runtime/releases)
- [Commits](https://github.com/dotnet/runtime/compare/v6.0.0...v7.0.1)

---
updated-dependencies:
- dependency-name: System.Text.Json
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump Microsoft.Extensions.Caching.Abstractions in /src

Bumps [Microsoft.Extensions.Caching.Abstractions](https://github.com/dotnet/runtime) from 5.0.0 to 7.0.0.
- [Release notes](https://github.com/dotnet/runtime/releases)
- [Commits](https://github.com/dotnet/runtime/compare/v5.0.0...v7.0.0)

---
updated-dependencies:
- dependency-name: Microsoft.Extensions.Caching.Abstractions
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump NSwag.MSBuild from 13.11.1 to 13.18.2 in /src

Bumps [NSwag.MSBuild](https://github.com/RicoSuter/NSwag) from 13.11.1 to 13.18.2.
- [Release notes](https://github.com/RicoSuter/NSwag/releases)
- [Changelog](https://github.com/RicoSuter/NSwag/blob/master/CHANGELOG.md)
- [Commits](https://github.com/RicoSuter/NSwag/commits/v13.18.2)

---
updated-dependencies:
- dependency-name: NSwag.MSBuild
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump Microsoft.Extensions.Http from 6.0.0 to 7.0.0 in /src

Bumps [Microsoft.Extensions.Http](https://github.com/dotnet/runtime) from 6.0.0 to 7.0.0.
- [Release notes](https://github.com/dotnet/runtime/releases)
- [Commits](https://github.com/dotnet/runtime/compare/v6.0.0...v7.0.0)

---
updated-dependencies:
- dependency-name: Microsoft.Extensions.Http
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

skrashevich committed:
Update KinopoiskUnofficialInfo.ApiClient.csproj

skrashevich committed:
Update .gitignore

skrashevich committed:
Merge branch 'master' of https://github.com/skrashevich/jellyfin-plugin-kinopoisk

skrashevich committed:
Merge branch 'master' of https://github.com/skrashevich/jellyfin-plugin-kinopoisk

skrashevich committed:
Update ApiModelExtensions.cs

skrashevich committed:
small build process changes

skrashevich committed:
Update publish.sh

skrashevich committed:
version 10.7.5.7

skrashevich committed:
version 10.7.5.7

skrashevich committed:
update publish script and build manifest

skrashevich committed:
version 10.8.9.0

skrashevich committed:
Update README.md

skrashevich committed:
Update manifest.json

skrashevich committed:
version 10.8.9.1

skrashevich committed:
version 10.8.9.1

skrashevich committed:
version 10.8.9.1

skrashevich committed:
version 10.8.9.1

skrashevich committed:
version 10.8.9.2

dependabot[bot] committed:
Bump Microsoft.NET.Test.Sdk from 16.5.0 to 17.5.0 in /src

Bumps [Microsoft.NET.Test.Sdk](https://github.com/microsoft/vstest) from 16.5.0 to 17.5.0.
- [Release notes](https://github.com/microsoft/vstest/releases)
- [Changelog](https://github.com/microsoft/vstest/blob/main/docs/releases.md)
- [Commits](https://github.com/microsoft/vstest/compare/v16.5.0...v17.5.0)

---
updated-dependencies:
- dependency-name: Microsoft.NET.Test.Sdk
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

LinFor commented:
@skrashevich привет. Я тебе вроде дал права на upstream, веди если есть желание. Права готов выдать любые.

skrashevich commented:
> @skrashevich привет. Я тебе вроде дал права на upstream, веди если есть желание. Права готов выдать любые.

Видел, поэтому и сделал этот бранч, дабы вмержить по-человечески: у нас немного разные build-системы :) Если что, в телеграм у меня юзернейм такой же, как и тут

skrashevich committed:
version 10.8.9.2

skrashevich committed:
Refactor Kinopoisk solution configuration platforms

skrashevich committed:
fix 10.8.9.2

skrashevich committed:
version 10.8.9.3

skrashevich committed:
Update package versions to use Microsoft.Extensions.Http and System.Text.Json version 6 instead of 7, and Microsoft.Extensions.Caching.Abstractions and Microsoft.Extensions.Logging version 6 instead of 7

skrashevich committed:
version 10.8.9.3

skrashevich committed:
WIP: Upgrade to .NET 8.0 and improve build scripts

- Upgraded all projects from .NET 6.0 to .NET 8.0.
- Updated GitHub Actions workflow to use .NET 8.0.
- Changed `publish.sh` script to check for `gsed` command, update version to 10.9.0.0, and use dotnet@8.
- Adjusted target ABI in various configurations and manifests to match the new version.
- Replaced `PersonType` with `PersonKind` in `ApiModelExtensions.cs` and adjusted profession mapping accordingly.
- Updated package references across all projects to be compatible with .NET 8.0.
- Modified `KinopoiskPluginServiceRegistrator` to match updated interface with `IServerApplicationHost`.
- General cleanup and consistency improvements in project files and build configurations.

skrashevich committed:
WIP: Upgrade to .NET 8.0 and improve build scripts

- Upgraded all projects from .NET 6.0 to .NET 8.0.
- Updated GitHub Actions workflow to use .NET 8.0.
- Changed `publish.sh` script to check for `gsed` command, update version to 10.9.0.0, and use dotnet@8.
- Adjusted target ABI in various configurations and manifests to match the new version.
- Replaced `PersonType` with `PersonKind` in `ApiModelExtensions.cs` and adjusted profession mapping accordingly.
- Updated package references across all projects to be compatible with .NET 8.0.
- Modified `KinopoiskPluginServiceRegistrator` to match updated interface with `IServerApplicationHost`.
- General cleanup and consistency improvements in project files and build configurations.

skrashevich committed:
refactor(kinopoisk-plugin): change return type of ToPersonType to PersonKind

skrashevich committed:
refactor(kinopoisk-plugin): change return type of ToPersonType to PersonKind

skrashevich committed:
feat(manifest): add version 10.9.0.0 of КиноПоиск plugin with new features

skrashevich committed:
Merge branch 'v10-9'

skrashevich committed:
Expand director category and add roles in mappings
#23

skrashevich committed:
v10.9.0.1

skrashevich committed:
Merge branch 'v10-9'
